### PR TITLE
pci: Add v1.0.1

### PR DESCRIPTION
**Pull request by simonjbeaumont**
Signed-off-by: Si Beaumont <simonjbeaumont@gmail.com>

### CODE REVIEW

**Comment by mseri** [comment]
Thanks!

File: packages/pci/pci.1.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  "ctypes" {>= "0.4"}
+  "ocamlfind" {build}
+  "ounit" {test}
+  "ocamlbuild" {build}
+]
+available: [ocaml-version >= "4.01.0" & os = "linux"]
+depexts: [
+  [["debian"] ["hwdata" "libpci-dev"]]
+  [["ubuntu"] ["hwdata" "libpci-dev"]]
+  [["centos"] ["hwdata" "pciutils-devel"]]
```
Comment (mseri) [inline packages/pci/pci.1.0.1/opam]: Can you please add
```
  [["alpine"] ["hwdata" "pciutils-dev"]]
```
to make it compatible also with alpine?

Comment (simonjbeaumont) [reply packages/pci/pci.1.0.1/opam]: Added in 13834dc

File: packages/pci/pci.1.0.1/opam [hunk 2]
```diff
@@ -30,6 +30,7 @@ depexts: [
   [["ubuntu"] ["hwdata" "libpci-dev"]]
   [["centos"] ["hwdata" "pciutils-devel"]]
   [["alpine"] ["hwdata" "pciutils-dev"]]
+  [["fedora"] ["hwdata" "pciutils-dev"]]
```
Comment (mseri) [inline packages/pci/pci.1.0.1/opam]: I think for fedora is `pciutil-devel`

Comment (simonjbeaumont) [reply packages/pci/pci.1.0.1/opam]: This is a PITA :)

**Comment by mseri** [comment]
It did work for me with fedora 27. Merging

**Comment by mseri** [comment]
Tank you very much, and apologies for how long it took ;)

**Pull request merged**